package.json

```diff
@@ -81,6 +81,7 @@
     "relay-cursor-paging": "^0.2.0",
     "request": "^2.67.0",
     "source-map-support": "^0.4.3",
+    "url-join": "^4.0.0",
```

Comment:
This is the only way I found to safely concatenate / build URLs from components.  The built-in URL modules's `resolve()` method was insufficient, as was `path.join()`

Reply:
@izakp - just fyi, `path` should only be for file-system paths, not urls. (There's also `path.resolve`)